recipes/sextractor/0003-name_changes.patch

```diff
@@ -0,0 +1,88 @@
+diff --git a/configure.ac b/configure.ac
+index 7ff6cce..57290bd 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -335,7 +335,7 @@ debian/changelog
+ doc/Makefile
+ doc/src/conf.py
+ man/Makefile
+-man/sex.1
++man/s_extractor.1
```

Comment:
Can I suggest `s-extractor` instead of `s_extractor`? In other words, a dash, not an underscore. Command-line tools with underscores are very bothersome and look funny.

Reply:
I also think plain `source-extractor` is totally fine. Bash completion helps us to not need short command names.

Reply:
The only hitch here is that autoconf doesn't seem to like dashes in the name. I originally tried `s-extractor` at https://github.com/conda-forge/staged-recipes/pull/8764/files#diff-4ac6155767839d568a1ba12ba5540148R71  and https://github.com/conda-forge/staged-recipes/pull/8764/files#diff-4ac6155767839d568a1ba12ba5540148R73, but the latter caused an error.

I'm a little afraid to go too far down the rabbit hole of autoconf....but fine with a different name.

Reply:
weird!

Reply:
Changed `s_extractor` to `s-extractor`....